### PR TITLE
Update content and convert to md  of Proxy [es]

### LINKED ISSUE #4164: 11. web/javascript/reference/global_objects/proxy/index.html
Requiere actualizar el contenido

[MDN Inglés](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
[MDN Español](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Proxy)

### PR DESCRIPTION
**Pull request by davbrito**
Closes #4164

### CODE REVIEW

File: files/es/web/javascript/reference/global_objects/proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,478 @@
+---
+title: Proxy
+slug: Web/JavaScript/Reference/Global_Objects/Proxy
+browser-compat: javascript.builtins.Proxy
+translation_of: Web/JavaScript/Reference/Global_Objects/Proxy
+---
+
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/proxy/index.md:7]: Una linea de mas aqui :)

```suggestion
```

File: files/es/web/javascript/reference/global_objects/proxy/index.md [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+};
+
+const proxy2 = new Proxy(target, handler2);
+```
+
+Aquí hemos provisto una implementación del manipulador
+{{jsxref("Global_Objects/Proxy/Proxy/get", "get()")}}, el cual intercepta los
+intentos de acceder a las propiedades del objeto envuelto.
+
+Las funciones manipuladores son llamadas a menudo _trampas_, probablemente
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/proxy/index.md:65]:
```suggestion
Las funciones manipuladoras son llamadas a menudo _trampas_, probablemente
```

File: files/es/web/javascript/reference/global_objects/proxy/index.md [hunk 3]
```diff
@@ -0,0 +1,478 @@
...
+
+{{Specifications}}
+
+## Compatibilidad con navegadores
+
+{{Compat}}
+
+## Véase también
+
+- [Presentaci;on de "Proxies are awesome" Brendan Eich presentation at JSConf](https://www.youtube.com/watch?v=sClk6aB_CPk)
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/proxy/index.md:476]:
```suggestion
- [Presentación de "Proxies are awesome" Brendan Eich presentation at JSConf](https://www.youtube.com/watch?v=sClk6aB_CPk)
```

File: files/es/web/javascript/reference/global_objects/proxy/index.md [hunk 4]
```diff
@@ -0,0 +1,478 @@
+---
+title: Proxy
+slug: Web/JavaScript/Reference/Global_Objects/Proxy
+browser-compat: javascript.builtins.Proxy
+translation_of: Web/JavaScript/Reference/Global_Objects/Proxy
+---
+
+{{JSRef}}
+
+El objecto `Proxy` permite crear un intermediario para otro objeto, el cual
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/proxy/index.md:10]:
```suggestion
El objeto `Proxy` permite crear un intermediario para otro objeto, el cual
```

File: files/es/web/javascript/reference/global_objects/proxy/index.md [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+console.log(products['Firefox']); // { name: 'Firefox', type: 'browser' }
+console.log(products['Chrome']); // undefined
+console.log(products.browser); // [{ name: 'Firefox', type: 'browser' }, { name: 'SeaMonkey', type: 'browser' }]
+console.log(products.types); // ['browser', 'mailer']
+console.log(products.number); // 3
+```
+
+### Un ejemplo con todas las `trampas`
+
+Para crear un ejemplo con la lista compreta de `trampas`, con motivos
```
Comment (Vallejoanderson) [inline files/es/web/javascript/reference/global_objects/proxy/index.md:404]:
```suggestion
Para crear un ejemplo con la lista completa de `trampas`, con motivos
```

**Comment by davbrito** [comment]
@Vallejoanderson muchas gracias por tu review 🙌🏽

**Review by Vallejoanderson — APPROVED** [review]

**Pull request merged**